content/source/docs/cloud/registry/index.html.md

```diff
@@ -1,19 +1,27 @@
 
-Terraform Cloud's private module registry works similarly to the [public Terraform Registry](/docs/registry/index.html) and helps you share [Terraform modules](/docs/language/modules/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a [configuration designer](/docs/cloud/registry/design.html) to help you build new workspaces faster.  
+Terraform Cloud's private registry works similarly to the [public Terraform Registry](/docs/registry/index.html) and helps you share [Terraform providers](/docs/language/providers/index.html) and [Terraform modules](/docs/language/modules/index.html) across your organization. It includes support for versioning, a searchable list of available providers and modules, and a [configuration designer](/docs/cloud/registry/design.html) to help you build new workspaces faster.
 
-You can add both private and public modules to the registry:
+> **Hands-on:** Try the [Add Public Modules to your Private Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules) tutorial and [Share Modules in the Private Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules) tutorials on HashiCorp Learn.
 
-- [Private modules](/docs/cloud/registry/publish.html) are hosted on the registry and are only available to members of that organization. In Terraform Enterprise, they are also available to other organizations that are configured to [share modules](/docs/enterprise/admin/module-sharing.html) with that organization.
-- [Public modules](/docs/cloud/registry/add.html) are automatically synchronized from the Terraform Registry where they are hosted. Public modules are not supported in Terraform Enterprise.
+## Public Providers and Modules
 
-You can use [Sentinel Policies](/docs/cloud/sentinel/index.html) to enforce rules about how members of your organization can use these modules. For example, you can mandate that all non-root modules in Terraform configurations must be private or public modules from your own private module registry. You can even require that all of those modules use recent versions with a policy like [use-recent-versions-from-pmr.sentinel](https://github.com/hashicorp/terraform-guides/blob/master/governance/third-generation/cloud-agnostic/http-examples/use-recent-versions-from-pmr.sentinel).
+[Public modules and providers](/docs/cloud/registry/add.html) are hosted on the public Terraform Registry and Terraform Cloud automatically synchronizes them to an organization's private registry. This lets you clearly designate which public providers and modules are recommended for the organization and makes their supporting documentation and examples centrally accessible.
+
+~> **Note:** Terraform Enterprise does not support public modules and providers.
```

Comment:
For these, I'm not exactly clear on what they mean. Are we telling users that TFE can't reach the public registry due to networking constraints, or is this more about support available from HashiCorp for public modules?

Reply:
ohh you know, I tried to make it more active and now I see why it was originally phrased the way it was. I just want to communicate that this functionality to add these things to your registry isn't available yet in TFE. 

Maybe "You cannot add public modules and providers to a private registry in Terraform Enterprise." or "Public and private modules are not supported in Terraform Enterprise." The second one is the original phrasing.

Reply:
I put back the original phrasing of "Public and private modules are not supported in Terraform Enterprise."  Lmk if we want to do something else here!